arcgis-ios-sdk-samples/Content Display Logic/Custom BarButtonItem/FavoritesBarButtonItem.swift

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+class FavoritesBarButtonItem: UIBarButtonItem {
+    var sample: Sample
+    
+    init(sample: Sample) {
+        self.sample = sample
+        super.init()
+        self.image = updateImage(isFavorite: sample.isFavorite)
+        self.target = self
+        self.action = #selector(FavoritesBarButtonItem.toggleIsFavorite)
```

Comment:
```suggestion
            action: #selector(toggleIsFavorite)
```

Reply:
Is there a typo here? There are build errors when I tried to apply the suggestion.

Reply:
Uh, yes, there is. The suggestion should have just removed the `FavoritesBarButtonItem.` part.